package.json

```diff
@@ -136,6 +136,7 @@
     "build-themes": "gulp style-compiler-themes",
     "build-themebuilder-assets": "gulp style-compiler-tb-assets",
     "dev": "gulp dev",
+    "dev-map": "gulp dev --sourceMap",
```

Comment:
I think we need one "dev" script.

Reply:
I've discussed it with Dima and proposed to rename tasks in such way
dev-map ->dev
dev -> dev-ie (Because it shuold be left only for the IE debugging)